nss-day-cohort-41/tabloidmvc-casino
# Ticket 21: Delete Comment

Ticket 21: Delete Comment 

Given a user is viewing a posts' list of comments they can click on the delete affordance. This will redirect a user to a new page that will ask for delete confirmation. A user can select to go back to list to return to the view of comments or the user can continue with deleting the comment. Upon deletion, the user will currently be redirected to the comment create form (fix for that later) The user can then return to the posts' comments and will see that comment has been deleted. 

Type of change
Adjustments made to Comment Repository, ICommentRepository, CommentController, & Details.cshtml.


Testing Instructions
Fetch my branch 'comments-delete'
You will have to have data in your SQL database to view categories
*I added....
SET IDENTITY_INSERT [Comment] ON
INSERT INTO [Comment] ([Id], [PostId], [UserProfileId], [Content], [Subject], [CreateDateTime])
VALUES (5, 1, 1, 'Comment One', 'Comment One', '09:45'), (2, 1, 1, 'Comment Two', 'Comment Two', '09:45'),
(3, 1, 1, 'Comment Three', 'Comment Three', '09:45'), (4, 1, 1, 'Comment Four', 'Comment Four', '09:45')
SET IDENTITY_INSERT [Comment] OFF;
as a new query... *
Run as Tabloid MVC and note that app should open a new browser.
Login in with established credentials.
Click on post and then details
Click on comment to visit a form for creation
Run through the delete process to test.
